docs/diagrams/IdClassDiagram.puml

```diff
@@ -5,8 +5,6 @@ skinparam arrowColor MODEL_COLOR
 skinparam classBackgroundColor MODEL_COLOR
 
 Class Person
```

Comment:
Might be a good idea to remove Person too and just show Id class alone with its fields.. because now it looks like the Person class is only made of Id (since there's the composition thing there)